backend/src/sql/list-proposals.sql

```diff
@@ -73,7 +82,7 @@ SELECT
     ) AS yes_votes,
     COALESCE(SUM(ldd_drep.amount) FILTER (WHERE voting_procedure.vote::text = 'No'), 0) + (
         CASE WHEN gov_action_proposal.type = 'NoConfidence' THEN
-            0
+            0`
```

Comment:
the  string literal sign seems to be a mistake?

Reply:
```suggestion
            0
```